### ISSUE TITLE
multipass find returns invalid launch values

### ISSUE THREAD

**Issue by hatched** [CanonicalLtd/multipass#188]
```
$ lsb_release -r
Release:	16.04
$ multipass --version
multipass  2018.2.1-38-g9f246b1
multipassd 2018.2.1-38-g9f246b1
$ multipass find
multipass launch …   Starts an instance of   Image version
----------------------------------------------------------
14.04                Ubuntu 14.04 LTS        20180222.1
   (or: t, trusty)
16.04                Ubuntu 16.04 LTS        20180306
   (or: default, lts, x, xenial)
17.10                Ubuntu 17.10            20180303
   (or: a, artful)
daily:18.04          Ubuntu 18.04 LTS        20180312
   (or: b, bionic, devel)
$ multipass launch -n bb 14.04
failed to launch: illegal value
$ multipass launch -n bb trusty
Launched: bb
```

**Comment by townsend2010** [comment]
I cannot reproduce this using both the source build nor the Multipass Snap in the Edge channel. Could you please try this again? Thanks!

**Comment by BryanQuigley** [comment]
I have a similar issue (but 14.04 works for me) with Bionic. (multipass --version 2018.4.1-15-g0cd4864)

$ multipass launch b
failed to launch: Too many images matching "b"                                  
$ multipass launch bionic
failed to launch: Unable to find an image matching "bionic"                     
$ multipass launch devel
failed to launch: Unable to find an image matching "devel"                      
$ multipass launch 18.04 #No one says this should work, but I'd like it
failed to launch: Unable to find an image matching "18.04"                      
$ multipass launch daily:18.04 #this works
Retrieving image: 5%

**Comment by Saviq** [comment]
@BryanQuigley the `daily:` bit is required for `devel`, otherwise you only get images from the released streams, which does not have devel, or 18.04 yet.

So if anything, your issue is our bad on documentation rather than @hatched's bug.

**Comment by hatched** [comment]
Sorry for the delay, I haven't been able to reproduce this lately with the most recent versions...maybe it's been repaired? Maybe it was just a poor error message and it meant something else?

Feel free to close this issue...
